### PR TITLE
docs: New landing page

### PR DESCRIPTION
**Pull request by dadrus**
## Checklist

- [x] I agree to follow this project's [Code of Conduct](../CODE_OF_CONDUCT.md).
- [x] I have read, and I am following this repository's [Contributing Guidelines](../CONTRIBUTING.md).
- [x] I have read the [Security Policy](../SECURITY.md).
- [x] I have updated the documentation.

## Description

When ready, it would update the current landing page with useful information and be hopefully appealing in sense of the layout.

### CODE REVIEW

File: docs/content/_index.adoc [hunk 1]
```diff
@@ -6,9 +6,51 @@ draft: false
...
 >}}
-Adopt Zero Trust architecture by authenticating, enriching and authorizing incoming HTTP requests at any level. No need to learn new protocols, authentication and authorization frameworks. You can still use them.
+Heimdall is a cloud native Identity Aware Proxy and Access Control Decision service, which can help you adopting Zero Trust architecture by authenticating, enriching and authorizing incoming HTTP requests at any level. No need to learn new protocols, authentication and authorization frameworks.
+<br><br>
+<a class="btn btn-primary btn-lg" type="button" href="link:{{< relref "/docs/getting_started/concepts.adoc" >}}">Get Started</a>
 {{< /seo >}}
 
+{{% seo title="Easy to integrate" %}}
+
+Use technics you are already familiar with
```
Comment (netthier) [inline docs/content/_index.adoc]: Should be "techniques"

File: docs/content/_index.adoc [hunk 2]
```diff
@@ -6,9 +6,51 @@ draft: false
...
+  rules:
+    - id: my_api_rule
+      match:
+        url: http://127.0.0.1:9090/api/<**>
+      execute:
+        - authenticator: keycloak
+        - authorizer: opa
+----
+
+The decision process can be controlled by each and every upstream service individually via rules, which heimdall can load from different sources, like e.g. from a `RuleSet` kubernetes resource.
```
Comment (netthier) [inline docs/content/_index.adoc]: Not sure if this is the best way to put this, the decision process isn't "controlled" by the upstream services, it's controlled by the user writing the rules.
Maybe something along the lines of "Create rules for each upstream and path you want to secure, loading them from a variety of sources such as Kubernetes custom resources, S3 buckets or regular files".
Also not sure about using the Kubernetes resource as the primary example, is it the most used configuration type?

File: docs/content/_index.adoc [hunk 3]
```diff
@@ -6,9 +6,51 @@ draft: false
...
+
+
+{{% /seo %}}
+
+{{% seo title="Reduces the cognitive load of your team" %}}
+
+By outsourcing authentication and authorization decisions to heimdall you can reduce the complexity of your code base, free resources and reduce the cognitive load of your team pretty much.
```
Comment (netthier) [inline docs/content/_index.adoc]: Not sure if "pretty much" is correct here, sounds a bit awkward. I'd either replace it with something like "substantially" or remove it.

File: docs/content/_index.adoc [hunk 4]
```diff
@@ -6,9 +6,51 @@ draft: false
...
+{{% /seo %}}
+
+{{% seo title="Feature rich" %}}
+
+* Use pipelines to compose reusable steps into the logic you need
+* Dynamically enrich requests with information like e.g. user data to help you make authorization decisions
+* Use Common Expression Language to implement complex authorization logic within a pipeline
+* Use existing authorization systems, like OpenFGA, Ory Keto, Open Policy Agent and many more.
+* Use existing authentication systems supporting OpenID Connect or OAuth2
+* Combine existing authentication systems
```
Comment (netthier) [inline docs/content/_index.adoc]: This point might be worth merging with the pipeline one as "combining existing authentication systems" is nothing more than composing authentication steps in a pipeline.
Not sure about a good way to put it though, maybe adding examples for the kinds of composition possible would be useful.

File: docs/content/_index.adoc (hunk 1 above)
Comment (dadrus) [reply docs/content/_index.adoc]: Thanks! Fixed

File: docs/content/_index.adoc (hunk 2 above)
Comment (dadrus) [reply docs/content/_index.adoc]: Replaced with your suggestion: "Create rules for each upstream and path you want to secure, loading them from a variety of sources such as Kubernetes custom resources, S3 buckets or regular files"

Comment (dadrus) [reply docs/content/_index.adoc]: > Also not sure about using the Kubernetes resource as the primary example, is it the most used configuration type?

Me neither. Maybe adding an example for e.g. file system is worth it as well and making it clickable e.g. as tabs or alike.

File: docs/content/_index.adoc (hunk 3 above)
Comment (dadrus) [reply docs/content/_index.adoc]: Yeah, you're right. Changed that based on your suggestion.

File: docs/content/_index.adoc (hunk 2 above)
Comment (netthier) [reply docs/content/_index.adoc]: Yeah, a comparison like that might be nice

File: docs/content/_index.adoc (hunk 3 above)
Comment (dadrus) [reply docs/content/_index.adoc]: Reasoning for adding that: I want to highlight that systems like heimdall can help reducing complexity. Actually, reducing is a wrong word. The complexity is there, but you can shift it, making it less problematic for the own code. Or even not having to deal with it at all. This already starts with the gained ability to decide how the service gets informed about the subject, and alike. All of that can make the implementation of the upstream service simpler and also much easier to test.

Comment (netthier) [reply docs/content/_index.adoc]: Definitely agree with the content and I'd argue that if you're using a microservice architecture you are actually actively reducing complexity by centralizing it in one place.

**Pull request merged**